lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -66,6 +66,23 @@ def repositories
   @repositories
 end
 
+def puppetfile
+  unless @puppetfile
+    @puppetfile = fixtures('puppetfile')
+    return nil if @puppetfile.empty?
+    require 'r10k/action/puppetfile/install'
```

Comment:
Either the gemspec needs to be updated to depend directly on `r10k` or this needs to catch `LoadError` exception and handle the case of `r10k` being not installed. (I would prefer the latter.)

I know @DavidS indicated that current versions of PDK have non-explicit dependencies on `r10k` through Litmus and Bolt, but PDK is definitely not the only usage of this gem and I'm also uncomfortable guaranteeing that PDK will always and forever include `r10k` going forward.